inst/tutorials/chapter4/mutlivariate_standalone.Rmd

```diff
@@ -90,23 +89,24 @@ output$reg_mul <- renderPlotly({
   scene = list(
     xaxis = axx,
     yaxis = axy,
-    zaxis = axz)
+    zaxis = axz, 
+    aspectmode = "cube")
   
   if ((a == a_true) && (b1 == b1_true) && (b2 == b2_true)){
     plot_ly(x = possible_x1, y = possible_x2, z = surf_matrix)  %>%
-    add_surface(name = "Regression Plane", hoverinfo = 'name', opacity = .75) %>%
+    add_surface(name = "Regression Plane", hoverinfo = 'name', opacity = .5, surfacecolor = rep('green', length(possible_x1))) %>%
     hide_colorbar()%>%
```

Comment:
for some reason, when i get the correct estimates (green points), the z-coordinate is no longer fixed. it's perfect while the slider is wrong.
can you check that?